src/javascript/binary/common_functions/account_opening.js

```diff
@@ -182,12 +182,36 @@ const AccountOpening = (() => {
         return validations;
     };
 
+    const handleAccountType = (e = {}) => {
+        if(e.target && e.target.value === 'ico') {
+            professionalClient.init(false, false, true);
+            $('h1').html(localize('ICO Account Opening'));
+        } else {
+            document.getElementById('fs_professional').setVisibility(0);
+            $('h1').html(localize('Real Money Account Opening'));
+        }
+    };
+
+    const accountTypeInit = (form_id) => {
+        $(form_id)
+            // Show account type options
+            .find('#fieldset_account_type')[0].setVisibility(1);
```

Comment:
no need for [0], can keep it as jquery object

Reply:
@negarn But `.show` on the object doesn't work.

Reply:
there is .setVisibility() for jquery too